txserver/main.go

```diff
@@ -99,3 +103,10 @@ func setup() *amqp.Channel {
 
 	return ch
 }
+
+func setupDB(ctx context.Context) (*mongo.Client, context.CancelFunc) {
```

Comment:
minor: we should have this fn in db.go file. We might add some more functions  in db.go later

Reply:
What other functions, I can't think of any atm. I believe we will only be using the mongo client interface as mentioned in the docs: https://pkg.go.dev/go.mongodb.org/mongo-driver@v1.8.3/mongo. Lmk if there's something else you're thinking about

Reply:
I was thinking of adding a fn called 'InsertEvent()' which will be a helper fn used by other handlers to save different logTypes to our collection. Maybe we will discover more along our way. db.go seems like a good home for these